site/docs/src/json/identity-providers/nintendo-response.json

```diff
@@ -0,0 +1,33 @@
+{
+  "identityProvider" : {
+    "applicationConfiguration" : {
+      "0ea8348a-6157-4ab0-b52c-fcc992bc8b85" : {
+        "createRegistration" : true,
+        "enabled" : true
+      }
+    },
+    "buttonText" : "Login with Nintendo",
+    "client_id" : "383376a2-4fb9-41b8-9823-288456add46c",
+    "client_secret" : "Xp5t7n9n66i4nhjp",
+    "debug" : false,
+    "emailClaim" : "email",
+    "enabled" : true,
+    "id" : "7764b5c7-165b-4e7e-94aa-02ebe2a0a5fb",
```

Comment:
This is not the correct Id, it should be `b0ac2e16-d4af-483e-98c8-7f6693610665`.

Reply:
I just thought it was a dummy value as an example. Didn't realize it needed to be something specific. I'll change it.